### PR TITLE
Introduce Charge SOM User Guide

### PR DESCRIPTION
**Pull request by lategoodbye**
This introducd the initial version of the Charge SOM User Guide, which only contains "Getting Started".

### CODE REVIEW

**Review by t-mon — COMMENTED** [review]
Is the black rectangle in the  [docs/source/_static/images/dc_charger_charge_som_setup.svg](https://github.com/chargebyte/chargebyte-docs/pull/23/files#diff-e4b9d8df26b17f083f3d23070274808004e4b528fb4185d6aed91a8a5e4bb843) intentional?

File: docs/source/conf.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 # -- Project information -----------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
-project = 'Charge Control C User Guide'
+project = 'Charge SOM User Guide'
 copyright = '2024, chargebyte GmbH'
```
Comment (t-mon) [inline docs/source/conf.py:10]: Maybe add 2025 already?

File: docs/source/getting_started.rst [hunk 2]
```diff
@@ -15,32 +15,27 @@ Setting Up the Hardware
...
+The following hardware components are required to set up the basic DC charger:
 
-- Charge Control C
+- Charge SOM Evaluation Kit
 - 12 V DC Power Supply
-- Power Contactor
-- IEC 62196 Type 2 three-phase EV charging socket outlet
 - Ethernet cable for SSH connection or USB to serial adapter for serial connection
-- IEC 62196 Type 2 EVSE Test Adapter (e.g. Metrel or Benning) to simulate the EV
+- DIN70121 or ISO15118-2 EV simulator (e.g. Charge Module S Evaluation board)
```
Comment (t-mon) [inline docs/source/getting_started.rst:23]: Maybe add a link to the product or to the product section

File: docs/source/getting_started.rst [hunk 3]
```diff
@@ -55,15 +50,14 @@ First Startup
...
 - The file system basically consists of three ext4 partitions. Two partitions are used as slots for
   the RAUC update process. The third partition is not touched by the RAUC update process and is usually
-  used for storing update bundles, logs, etc. For more information about the firmware in general,
-  firmware updates and the partition layout, see the :ref:`firmware.rst` chapter.
-- After connecting the Charge Control C to the power supply, the U-Boot bootloader starts the
+  used for storing update bundles, logs, etc.
+- After connecting the Charge controller to the power supply, the U-Boot bootloader starts the
   currently active slot managed by RAUC.
-- The LED status indicators on the Charge Control C provide information about the current status of
+- The LED status indicators on the Charge controller provide information about the current status of
```
Comment (t-mon) [inline docs/source/getting_started.rst:60]: Just a tought: maybe write `on the SOM` or `on the carrier board` to be more specific which LEDs

File: docs/source/getting_started.rst [hunk 4]
```diff
@@ -72,90 +66,57 @@ Here are some key points about the boot process of the Charge Control C:
 Understanding LED Status Indicators
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Now you can plug in the Charge Control C to the power supply. The LED status indicators on the
-Charge Control C provide information about the current status of the boot process. The following
-table shows the meaning of the LED status indicators:
+Now you can plug in the Charge SOM Evaluation Kit to the power supply. The LED status indicator on the
+carrier board provide information about the current status of the boot process. The following table
```
Comment (t-mon) [inline docs/source/getting_started.rst:70]: As mentioned in the comment above, maybe keep the wording when describing a location of something.

File: docs/source/getting_started.rst [hunk 5]
```diff
@@ -166,22 +127,22 @@ other over the interfaces. The following figure illustrates how the EVerest modu
...
-The configuration of the CbTarragonDriver module can be found in "/usr/libexec/everest/modules/CbTarragonDriver"
+need to be adapted. The hardware related tasks are mainly handled by the CbChargeSOMDriver module.
+The configuration of the CbChargeSOMDriver module can be found in "/usr/libexec/everest/modules/CbChargeSOMDriver"
 directory.
 
 Each module has a specific configuration file. This file is called "manifest.yaml" and is stored 
 in the main directory of the module.
 Here you can also see all other configuration parameters of the respective module. 
-Now please type "less /usr/libexec/everest/modules/CbTarragonDriver/manifest.yaml" to see the
+Now please type "less /usr/libexec/everest/modules/CbChargeSOMDriver/manifest.yaml" to see the
```
Comment (t-mon) [inline docs/source/getting_started.rst:145]: Maybe rephrase so it does not sound like a request, more like an option?

In order to see the content of the configuration file you can take a look using  "less /usr/libexec/everest/modules/CbChargeSOMDriver/manifest.yaml" ...

File: docs/source/getting_started.rst [hunk 6]
```diff
@@ -230,98 +192,178 @@ The EVerest log should look like this:
...
+   2024-12-13T15:52:02.223254+0100 chargesom manager[261]: [INFO] auth:Auth        :: Providing authorization to connector#1
+   2024-12-13T15:52:02.960278+0100 chargesom manager[263]: [INFO] connector:EvseM  :: EVSE IEC Set PWM On (5.000000074505806%) took 0 ms
+   2024-12-13T15:52:02.962409+0100 chargesom manager[262]: [INFO] bsp:CbChargeSOM  :: handle_pwm_on: Setting new duty cycle of 5.00%
+   2024-12-13T15:52:03.105628+0100 chargesom manager[261]: [INFO] auth:Auth        :: Result for token: DEADBEEF: ACCEPTED
+   2024-12-13T15:52:03.155035+0100 chargesom manager[263]: [INFO] connector:EvseM  :: EVSE IEC EIM Authorization received
+   2024-12-13T15:52:03.155499+0100 chargesom manager[263]: [INFO] connector:EvseM  :: EVSE IEC Transaction Started (0 kWh)
+   2024-12-13T15:52:03.156415+0100 chargesom manager[263]: [INFO] connector:EvseM  :: EVSE IEC DC mode. We are in 5percent mode so we can continue without further action.
+   2024-12-13T15:52:03.156895+0100 chargesom manager[263]: [INFO] connector:EvseM  :: EVSE IEC Charger state: Wait for Auth->PrepareCharging
+   
+After that the EV simulator should establish a powerline connection to the Charge SOM via SLAC.
```
Comment (t-mon) [inline docs/source/getting_started.rst:289]: After that`,` the EV simulator

File: docs/source/getting_started.rst [hunk 7]
```diff
@@ -230,98 +192,178 @@ The EVerest log should look like this:
...
+   2024-12-13T15:52:04.561486+0100 chargesom manager[267]: [INFO] evse_slac:EvseS  :: Session (run_id=D895B977488179F2, ev_mac=00:01:87:05:A0:84): Finalize sounding, sending CM_ATTEN_CHAR_IND
+   2024-12-13T15:52:04.577453+0100 chargesom manager[267]: [INFO] evse_slac:EvseS  :: Session (run_id=D895B977488179F2, ev_mac=00:01:87:05:A0:84): received CM_ATTEN_CHAR_RSP, going to substate WAIT_FOR_SLAC_MATCH
+   2024-12-13T15:52:04.615447+0100 chargesom manager[267]: [INFO] evse_slac:EvseS  :: Session (run_id=D895B977488179F2, ev_mac=00:01:87:05:A0:84): Received CM_SLAC_MATCH_REQ, sending CM_SLAC_MATCH_CNF -> session complete
+   2024-12-13T15:52:04.616909+0100 chargesom manager[267]: [INFO] evse_slac:EvseS  :: Entered Matched state
+   2024-12-13T15:52:04.637408+0100 chargesom manager[263]: [INFO] connector:EvseM  :: EVSE ISO SLAC MATCHED
+   2024-12-13T15:52:04.661360+0100 chargesom manager[263]: [INFO] connector:EvseM  :: EVSE ISO D-LINK_READY (true)
+
+Now the EV simulator discovers the V2G service of the Charge SOM and establish a TCP connection.
```
Comment (t-mon) [inline docs/source/getting_started.rst:325]: establishes

File: docs/source/getting_started.rst [hunk 8]
```diff
@@ -230,98 +192,178 @@ The EVerest log should look like this:
...
+   2024-12-13T15:52:09.049641+0100 chargesom manager[270]: [INFO] iso15118_charge  :: Parameter-phase started
+   2024-12-13T15:52:09.049641+0100 chargesom manager[270]: [INFO] iso15118_charge  :: Selected energy transfer mode: DC_extended
+   2024-12-13T15:52:09.057571+0100 chargesom manager[263]: [INFO] connector:EvseM  :: Received EV maximum limits: {
+   2024-12-13T15:52:09.057571+0100 chargesom manager[263]:     "dc_ev_maximum_current_limit": 0.0,
+   2024-12-13T15:52:09.057571+0100 chargesom manager[263]:     "dc_ev_maximum_voltage_limit": 0.0
+   2024-12-13T15:52:09.057571+0100 chargesom manager[263]: }
+   2024-12-13T15:52:09.058568+0100 chargesom manager[263]: [INFO] connector:EvseM  ::                                     CAR ISO V2G ChargeParameterDiscoveryReq
+   2024-12-13T15:52:09.147996+0100 chargesom manager[263]: [INFO] connector:EvseM  :: EVSE ISO V2G ChargeParameterDiscoveryRes
+
+Congratulations! You have successfully established a charging session with the EVerest
```
Comment (t-mon) [inline docs/source/getting_started.rst:366]: is the `!` intentional?

**Review by t-mon — COMMENTED** [review]
Otherwise lgtm

**Comment by lategoodbye** [comment]
> Is the black rectangle in the [docs/source/_static/images/dc_charger_charge_som_setup.svg](https://github.com/chargebyte/chargebyte-docs/pull/23/files#diff-e4b9d8df26b17f083f3d23070274808004e4b528fb4185d6aed91a8a5e4bb843) intentional?

No, this wasn't intentend.

File: docs/source/getting_started.rst (hunk 3 above)
Comment (lategoodbye) [reply docs/source/getting_started.rst:60]: Yes, i will changed it to carrier board. I also suggested Simon to add it into the graphic to be more clear.

File: docs/source/getting_started.rst (hunk 5 above)
Comment (lategoodbye) [reply docs/source/getting_started.rst:145]: Sorry, just copy & paste from the original document. The complete section is crap. Will try to fix it.

File: docs/source/getting_started.rst (hunk 8 above)
Comment (lategoodbye) [reply docs/source/getting_started.rst:366]: I took over this from the Charge Control C. So yes

File: docs/source/getting_started.rst (hunk 4 above)
Comment (lategoodbye) [reply docs/source/getting_started.rst:70]: I hope this is better now

File: docs/source/getting_started.rst (hunk 5 above)
Comment (lategoodbye) [reply docs/source/getting_started.rst]: Okay, i tried to improve the section so that users are not tempt to modify the manifest file.
